src/default_parameters.jl

```diff
@@ -109,7 +109,8 @@ The default values of the keywords define the default model setup.
 
     # INITIAL CONDITIONS
     seed::Int = abs(rand(Int))          # a random seed that's used in initialize_speedy for the global RNG
-    initial_conditions::Symbol=:barotropic_vorticity    # :rest, :barotropic_vorticity or :restart
+    initial_conditions::Union{Symbol, Vector}=:barotropic_vorticity    # :rest, :barotropic_vorticity, :restart
```

Comment:
Another problem I see here is that with `output=true` the parameters struct is printed to `parameters.txt` such that one can easily trace back what parameters were used in a given simulation. I'm not sure what happens if a big array is part of the parameters struct. This also applies to setting the vertical levels manually, which I haven't checkedn. Those ones, ideally, should be written into that .txt file (as just a vector) but entire vectors or matrices obviously not...